BritneyS/portfolio-site-gatsby
# Edit metadata

## What does this PR do :question:
This PR edits the title, description, and favicon for the site.


## Any packages added :package:
N/A


## How to test it :microscope:
1. Checkout this branch
2. Run the site with `gatsby develop` and navigate to `http://localhost:8000/` (or the URL given in the terminal if you have another service running on that port)
3. Confirm that the tab has the title `Home | Britney Smith: A Software Developer For The Web And iOS`
4. Confirm that the favicon (tab icon) is ✨ 


## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
<img width="456" alt="Screen Shot 2020-01-18 at 5 22 13 PM" src="https://user-images.githubusercontent.com/8409475/72671374-235a4a00-3a17-11ea-95ad-93e97d5fdfd5.png">
